src/character.cpp

```diff
@@ -482,6 +482,7 @@ Character::Character() :
     }
     // Only call these if game is initialized
     if( !!g && json_flag::is_ready() ) {
+        set_cardio_acc( base_bmr() / 2 );
```

Comment:
Following up on my comment:

> I believe this large random value may be because `cardio_acc` is uninitialized in the `Character` constructor.

Here is where the `cardio_acc` initialization should occur - but it is gated by `json_flag::is_ready`, which for whatever reason is false when the `Character` constructor is called. I believe the reason for stipulating this condition is that `base_bmr()` needs `metabolic_rate_base()`, which reads from `game_balance.json` the setting for `PLAYER_HUNGER_RATE` (default 1.0) - so it needs this `is_ready()` flag to be true in order to read that balance setting.

If we can find a way to initialize `cardio_acc` to a reasonable value without depending on `base_bmr()`, that should resolve the problem with new characters having wildly random cardio values. Or, find a way to ensure `json_flag::is_ready()` is true before we reach this constructor.

Reply:
I will figure out a solution for this in the new PR.